portfolio/pom.xml

```diff
@@ -23,11 +23,6 @@
       <version>4.0.1</version>
       <scope>provided</scope>
     </dependency>
-    <dependency>
-      <groupId>com.google.code.gson</groupId>
-      <artifactId>gson</artifactId>
-      <version>2.8.6</version>
-    </dependency>
```

Comment:
This may be due to some branching issues, make sure you're not deleting this code :)

Reply:
when i merged it showed up twice so i deleted it here! this dependency is on line 31 now!